.github/workflows/build_wheels.yml

```diff
@@ -134,7 +138,7 @@ jobs:
 
   verify-python-wheels:
     runs-on: ${{ matrix.os }}
-    needs: [build-python-wheel, build-source-distribution]
+    needs: [build-python-wheel, build-source-distribution, get-version]
```

Comment:
Why is this change needed?

Reply:
we need to give the job access the release version which means we must add it to needs